pipes/rules/metagenomics.rules

```diff
@@ -50,54 +50,97 @@ method_props = {
     }
 }
 
+taxfiles = [
+                'gi_taxid_nucl.dmp',
+                'gi_taxid_prot.dmp',
+                'names.dmp',
+                'nodes.dmp'
```

Comment:
Merged is a list of taxonomy nodes that got merged into another node. It's useful to process this somehow by translating the taxids to ensure a bit extra durability keeping taxonomy and database fastas in sync. Otherwise every database and every version would need their own copy of the taxonomy directory.